### PR TITLE
bump-up-to-v0.4.0-beta.1

### PR DESCRIPTION
**Pull request by tuommaki**

### CODE REVIEW

**Comment by fiunchinho** [comment]
This PR contains:
- Using a static name for the app manifests, so that they don't change when releasing a new version.
- Merging webhook and reconcilers, as they got merged by upstream in latest releases

File: helm/cluster-api-control-plane/values.yaml [hunk 1]
```diff
@@ -1,7 +1,16 @@
-name: cluster-api
+name: cluster-api-control-plane
+
+registry:
+  domain: quay.io
+
 image:
   name: giantswarm/kubeadm-control-plane-controller
-  tag: v0.3.22-gs3
+  tag: fa3378e532147820227c7b7a887d9e7ab54fc2ff
```
Comment (tuommaki) [inline helm/cluster-api-control-plane/values.yaml:8]: I used `v0.4.0` here

**Comment by tuommaki** [comment]
I think you also need this: https://github.com/giantswarm/cluster-api-bootstrap-provider-kubeadm-app/pull/23/commits/bc0f950852d36c54b205757652f3f65cca0f447c

**Comment by fiunchinho** [comment]
Added

**Comment by fiunchinho** [comment]
Superseeded by https://github.com/giantswarm/cluster-api-app

**Pull request closed without merging**